samples/samples/admin/samples.py

```diff
@@ -61,16 +61,94 @@ def create_instance(instance_id):
 # [END spanner_create_instance]
 
 
+# [START spanner_create_instance_with_processing_units]
+def create_instance_with_processing_units(instance_id, processing_units):
+    """Creates an instance."""
+    spanner_client = spanner.Client()
+
+    config_name = "{}/instanceConfigs/regional-us-central1".format(
+        spanner_client.project_name
+    )
+
+    request = spanner_instance_admin.CreateInstanceRequest(
+        parent="projects/{}".format(spanner_client.project),
+        instance_id=instance_id,
+        instance=spanner_instance_admin.Instance(
+            config=config_name,
+            display_name="This is a display name.",
+            processing_units=processing_units,
+            labels={
+                "cloud_spanner_samples": "true",
+                "sample_name": "snippets-create_instance_with_processing_units",
+                "created": str(int(time.time())),
+            },
+        ),
+    )
+
+    operation = spanner_client.instance_admin_api.create_instance(request=request)
+
+    print("Waiting for operation to complete...")
+    instance = operation.result(OPERATION_TIMEOUT_SECONDS)
+
+    print(
+        "Created instance {} with {} processing units".format(
+            instance_id, instance.processing_units
+        )
+    )
+
+
+# [END spanner_create_instance_with_processing_units]
+
+
+# [START spanner_create_database]
+def create_database(instance_id, database_id):
+    """Creates a database and tables for sample data."""
+    spanner_client = spanner.Client()
+    instance = spanner_client.instance(instance_id)
+
+    request = spanner_database_admin.CreateDatabaseRequest(
+        parent=instance.name,
+        create_statement=f"CREATE DATABASE `{database_id}`",
+        extra_statements=[
+            """CREATE TABLE Singers (
+            SingerId     INT64 NOT NULL,
+            FirstName    STRING(1024),
+            LastName     STRING(1024),
+            SingerInfo   BYTES(MAX),
+            FullName   STRING(2048) AS (
+                ARRAY_TO_STRING([FirstName, LastName], " ")
+            ) STORED
+        ) PRIMARY KEY (SingerId)""",
+            """CREATE TABLE Albums (
+            SingerId     INT64 NOT NULL,
+            AlbumId      INT64 NOT NULL,
+            AlbumTitle   STRING(MAX)
+        ) PRIMARY KEY (SingerId, AlbumId),
+        INTERLEAVE IN PARENT Singers ON DELETE CASCADE""",
+        ],
+    )
+
+    operation = spanner_client.database_admin_api.create_database(request=request)
```

Comment:
Quick check whether we are okay to not send `google-cloud-resource-prefix` header metadata as part of request. We do this for our handwritten layer currently.

Reply:
Yes we are okay to not send `google-cloud-resource-prefix`
Shared the artifacts offline to you.